### ISSUE TITLE
Absolute paths not supported

### ISSUE THREAD

**Issue by MartinRamm** [studioLaCosaNostra/publish-aab-google-play#3]
Given I use an absolute path for the `--keyFile` parameter, it prefixes the path with the current working directory.

For example:

```
mkdir -p /test/workingdir
cd /test
echo '{...}' > keyfile
cd /test/workingdir
npx deploy-aab-google-play --keyFile /test/keyfile ...
```
Output:
```
ENOENT: no such file or directory, open '/test/workingdir/test/keyfile'
```

**Comment by yamaha252** [comment]
The same problem. That's because it [adds](https://github.com/studioLaCosaNostra/publish-aab-google-play/blob/master/src/bin.ts#L25) workdir to the path 🤷‍♂️

**Comment by AnzeBlaBla** [comment]
This fork solves the issue:
https://github.com/PowerOfDark/publish-aab-google-play
It'd be cool if @studioLaCosaNostra updated the package.

**Comment by rayros** [comment]
Fixed in v1.1.6
